mint-auditor/migrations/2022-05-24-214504_create_initial/up.sql

```diff
@@ -3,7 +3,7 @@ CREATE TABLE block_audit_data (
     -- Diesel requires having a primary key and sqlite doesn't allow 64 bit primay keys, so even though
     -- we would've wanted to use the block_index for that we can't.
     -- Must be nullable for auto-increment: https://www.sqlite.org/autoinc.html
-    id INT PRIMARY KEY,
+    id INTEGER PRIMARY KEY,
```

Comment:
Turns out Sqlite3 is silly and doesn't enable auto-increment for `INT PRIMARY KEY` columns, they have to be `INTEGER PRIMARY KEY` 🤷‍♂️